common/predictive-text/package.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "keyman-lmlayer-prototype",
-  "version": "0.0.0",
+  "name": "keyman-lmlayer",
+  "version": "14.0.0",
   "description": "Keyman/Predictive Text integration layer",
   "main": "build/index.js",
   "scripts": {
     "karma": "karma",
     "mocha": "mocha",
     "test": "bash ./unit_tests/test.sh",
-    "tsc": "tsc",
-    "install": "npm link @keymanapp/lexical-model-types"
```

Comment:
This actually fixes an issue in modern versions of Node/npm, so I welcome this addition! 👍